### PR TITLE
Lizzers and plasmamen can now be servants of Ratvar

### LINKED ISSUE #31226: clock cult: lizards spawn as human with lizard name on reebe
[Directions]: # (Include the Round ID from the Status panel or retrieve it from https://atlantaned.space/newSS13tools/round.php ! If you believe the issue to be caused by a testmerge [OOC tab -> Show Server Revision], report it in the pull request's comment section instead. Explain your issue in detail, including the steps to reproduce it.)

[For Admins]: # (Oddities induced by var-edits and other admin tools are not necessarily bugs. Verify that your issues occur under regular circumstances before reporting them.)

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis and Y0SH1_M4S73R
fix: Servants of Ratvar now spawn in as the actual race set on their preferences, with plasmamen getting the gear they need to not immediately die.
/:cl:

During the rework, this was comparatively a fairly low-priority thing since the bandaid fix of just making all servants human worked equally well. Now that it's in and I have time buffer the rough edges, this seems like a fairly good step.

Servants of Ratvar now spawn in with their chosen race instead of being set to human. Plasmamen get appropriate gear for surviving the atmosphere.

Fixes #31226
Also fixes an undocumented issue where the `Naked` outfit gave the target the servant loadout instead.

### CODE REVIEW

**Comment by ChangelingRain** [comment]
Make sure clockwork armor doesn't autoremove the helmet, or it'll kill plasmamen.

**Comment by KorPhaeron** [comment]
Did plasmamen even exist in the time period that ratvar and friends got locked up in their own dimension?

**Comment by Xhuis** [comment]
@KorPhaeron The canon here isn't "humans that got sealed away with Ratvar" but I'm not sure exactly *what* it is otherwise. I think it's best left up to interpretation but I'm content with passing it off as crewmembers that were nabbed from a different station and brought to Reebe (similar to what happens with existing crew!)

**Comment by KorPhaeron** [comment]
Ratvar hopping planet to planet station to station and pulling material and people into Rebee like a yellow greygoo virus building up for the final conflict is my headcanon now

Fits nicely with people who like to imagine all rounds exist within the same universe

**Comment by Xhuis** [comment]
@ChangelingRain It looks like that already works due to space helmets being in the "don't replace this armor" typecache.

File: code/game/gamemodes/clock_cult/clock_cult.dm [hunk 1]
```diff
@@ -246,13 +246,25 @@ Credit where due:
...
 
 /datum/outfit/servant_of_ratvar/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
 	var/obj/item/card/id/W = H.wear_id
 	W.assignment = "Assistant"
 	W.access += ACCESS_MAINT_TUNNELS
 	W.registered_name = H.real_name
 	W.update_label()
+	if(plasmaman) //If we need to breathe from the plasma tank, we should probably start doing that
+		H.internal = H.get_item_for_held_index(2)
+		H.update_internals_hud_icon(1)
```
Comment (Cyberboss) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: dont call this if visualsOnly

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Pull request merged**